CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### Unreleased
+   * Strings: Rename `flip` to `reverse`. `flip` is deprecated and will be
+     removed in one release.
```

Comment:
I mean in the current version++. Reworded to 'the next release'. 

In theory every addition of a new top-level identifier is a potentially breaking change. Not sure if it's worth calling them out as such though. Added a bit more detail to the changelog.